### PR TITLE
fix for broken link, typo, and sample code

### PR DESCRIPTION
**Pull request by LiYing2010**
fix for broken link, typo, and sample code

### CODE REVIEW

File: docs/topics/runners/dokka-maven.md [hunk 1]
```diff
@@ -586,7 +586,7 @@ Below you can see all the possible configuration options applied at the same tim
         <suppressInheritedMembers>false</suppressInheritedMembers>
         <offlineMode>false</offlineMode>
         <sourceDirectories>
-            <dir>src</dir>
+            <dir>${project.basedir}/src</dir>
```
Comment (IgnatBeresnev) [inline docs/topics/runners/dokka-maven.md:589]: Can you please explain why this particular change is needed? 

A [recent PR](https://github.com/Kotlin/dokka/pull/3046/commits/40579e51027121864a05612b5fd52585d33f396e#diff-f812b435573bdfbf70feb8b89e12da5a618472d79ef4cbb1dbf1a6229bd968a6L410-R410) gave me the impression that paths like `${project.basedir}/src` are problematic on windows

**Review by IgnatBeresnev — APPROVED** [review]
Wow, thank you for noticing and fixing it!

File: docs/topics/runners/dokka-maven.md (hunk 1 above)
Comment (LiYing2010) [reply docs/topics/runners/dokka-maven.md:589]: here I fixed is `sourceDirectories`, and the PR you mentioned fixed `sourceLinks`, they are different

(1) about `sourceLinks`, according to doc here: https://kotlinlang.org/docs/dokka-maven.html#449338f2
```
The path must be relative to the root of the current module.
```
it must be a **relative** path, so the PR you mentioned is correct

(2) about `sourceDirectories`, according to doc here: https://kotlinlang.org/docs/dokka-maven.html#1be4d68
```
The source code roots to be analyzed and documented. Acceptable inputs are directories and individual .kt / .java files.

Default: {project.compileSourceRoots}
```

it does not mention about **relative** path or **absolute** path, but `Default` value looks like **absolute** path

and also, in doc here: https://kotlinlang.org/docs/dokka-maven.html#general-configuration
the sample is:
```
        <sourceDirectories>
            <dir>${project.basedir}/src</dir>
        </sourceDirectories>
```
it's also **absolute** path

Comment (LiYing2010) [reply docs/topics/runners/dokka-maven.md:589]: BTW, the PR you mentioned fixed `sourceLinks`, but only in `Source link configuration` section
in this PR, I fixed the same problem in `Complete configuration` section

see: https://github.com/Kotlin/dokka/pull/3532/files#diff-f812b435573bdfbf70feb8b89e12da5a618472d79ef4cbb1dbf1a6229bd968a6R620

**Comment by LiYing2010** [comment]
hi @IgnatBeresnev 
thank you very much for approving this PR
could you please merge it?

**Comment by IgnatBeresnev** [comment]
Thank you once again!

**Pull request merged**